### PR TITLE
better examples package errors

### PR DESCRIPTION
**Pull request by acunniffe**
## 🍗 Description
_What does this PR do? Anything folks should know?_

## 📚 References
_Links to relevant docs (Notion, Twist, GH issues, etc.), if applicable._

## 👹 QA
_How can other humans verify that this PR is correct?_

### CODE REVIEW

**Comment by acunniffe** [comment]
Merging for a customer, small surface area

File: projects/standard-rulesets/src/examples/requireValidExamples.ts [hunk 1]
```diff
@@ -48,7 +48,13 @@ export function validateSchema(
   const result = schemaCompiled(example);
 
   if (!result) {
-    const error = `- ${ajv.errorsText(schemaCompiled.errors, {
+    schemaCompiled.errors?.forEach((error) => {
+      if (error.keyword === 'additionalProperties') {
+        error.message = `must NOT have additional property '${error.params.additionalProperty}'`;
```
Comment (austen-herbst) [inline projects/standard-rulesets/src/examples/requireValidExamples.ts:53]: Nice!

**Review by austen-herbst — APPROVED** [review]

File: projects/standard-rulesets/src/examples/requireValidExamples.ts [hunk 2]
```diff
@@ -48,7 +48,13 @@ export function validateSchema(
   const result = schemaCompiled(example);
 
   if (!result) {
-    const error = `- ${ajv.errorsText(schemaCompiled.errors, {
+    schemaCompiled.errors?.forEach((error) => {
```
Comment (niclim) [inline projects/standard-rulesets/src/examples/requireValidExamples.ts:51]: I think we've got similar logic here https://github.com/opticdev/optic/blob/main/projects/openapi-io/src/validation/validator.ts#L67-L78

Do we want to move this into an `openapi-utilities` folder and report on AJV errors consistently?

**Review by niclim — APPROVED** [review]

File: projects/standard-rulesets/src/examples/requireValidExamples.ts (hunk 2 above)
Comment (acunniffe) [reply projects/standard-rulesets/src/examples/requireValidExamples.ts:51]: I looked into that, but it's very coupled with the sourcemap stuff. I think this is right for now.

**Pull request merged**